tests/test_tutorial/test_body/test_tutorial001.py

```diff
@@ -206,11 +206,14 @@ def test_post_broken_body(client: TestClient):
             "detail": [
                 {
                     "type": "json_invalid",
-                    "loc": ["body", 1],
-                    "msg": "JSON decode error",
-                    "input": {},
+                    "loc": ["body", 1, 2],
+                    "msg": "JSON decode error - Expecting property name enclosed in double quotes at line 1, column 2",
```

Comment:
```suggestion
                    "loc": ["body", 1],
                    "msg": "JSON decode error",
```

I think we don't need this duplication - having `error`, `line` and `column` in `ctx` should be enough.

Also, the semantic of `loc` field seems to be different - it should be indexes of list or dict